templates/header.html

```diff
@@ -38,12 +38,10 @@
           </ul>
         </div>
       {% else %}
-        <!-- <div class="g-signin2" data-onsuccess="onSignIn"></div> -->
-        <div id="gSignInWrapper">
-          <div id="gSignInCustomBtn" class="customGPlusSignIn">
-              <span class="gSignInIcon"></span>
-          </div>
+        <div id="g_id_onload" data-client_id="{{google_sign_in_client_id}}" data-ux_mode="redirect"
+          data-login_uri="${window.location.hostname + '/login'}">
```

Comment:
Replace this with [data-callback](https://developers.google.com/identity/gsi/web/reference/html-reference#data-callback)